mixer/cmd/root.go

```diff
@@ -85,16 +85,17 @@ var rootCmdFlags = struct {
 }{}
 
 type initCmdFlags struct {
-	all         bool
+	alllocal    bool
+	allupstream bool
 	clearver    int
 	mixver      int
+	localrpms   bool
```

Comment:
Optional: stick to the Go naming conventions, we'll fix the other names as we touch the code.

allLocal
allUpstream
localRPMs

Reply:
I greatly prefer that, I was just matching the rest of the code.